api/app.py

```diff
@@ -29,21 +34,32 @@ def serve():
     # be relative to the root of the repo, NOT to this file. You will likely need to specify the working directory
     # as the root of this repo # when you run this file in your IDE
     app.register_blueprint(user_bp, url_prefix="/authors")
-    app.register_blueprint(admin_bp, url_prefix="/admin")
+    app.register_blueprint(nodes_bp, url_prefix="/nodes")
+    app.register_blueprint(actions_bp, url_prefix="/admin/action")
     app.register_blueprint(swaggerui_blueprint, url_prefix="/docs")
 
     app.config.from_object("api.config.Config")
 
     if testing_env:
         app.config.update({"SQLALCHEMY_DATABASE_URI": "sqlite:///testing.db"})
 
+    basic_auth = APIAuth()
+    basic_auth.init_app(app)
     db.init_app(app)
     bcrypt.init_app(app)
     login_manager.init_app(app)
 
+    # admin views
+    app.config.update({"FLASK_ADMIN_SWATCH": "cerulean", "BASIC_AUTH_FORCE": APIConfig.is_API_protected})
```

Comment:
if we unconditionally set these, can these simply be set in `config.py` as properies?